SocketIOClientSwift/SocketEngine.swift

```diff
@@ -350,8 +350,10 @@ public final class SocketEngine: NSObject, WebSocketDelegate {
     // We had packets waiting for send when we upgraded
     // Send them raw
     private func flushWaitingForPostToWebSocket() {
-        for msg in postWait {
-            ws?.writeString(msg)
+        if let ws = ws {
```

Comment:
There's no need to break this out. `ws?.writeString(msg)` won't crash if ws is nil